config/routes.rb

```diff
@@ -53,4 +55,6 @@
   #     # (app/controllers/admin/products_controller.rb)
   #     resources :products
   #   end
+
+  root 'indexes#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.